source/includes/_introduction.md.erb

```diff
@@ -0,0 +1,16 @@
+# Introduction
+
+The SnapChance API is developed to support entertainment games such as scratch card game which gives a player an opportunity to win money.
+
+The API is secured with JWT authentication and the requests are authorized for the users with valid credentials. The user must authenticate with API and get a JWT token; which can be used to access all secured endpoints. The API has build-in integration with NETs for BankID verification and payment process.
+
+#### SnapChance API supports users with three roles:
+- <b>Regulator</b> - Create and manage the licenses.
+- <b>Operator/Admin</b> - Create and manage the lotteries.
+- <b>Player</b> - Add funds to the wallet; purchases and play tickets.
+
+Base URLs:
+
+* <a href="https://api-stg3.snapchance.no">https://api-stg3.snapchance.no</a>
+
+Web: <a href="https://snapchance.no">SnapChance</a>
```

Comment:
This is just a plain copy of the original text. I will update this in a separate PR

Reply:
👍

Reply:
@dkaga note that this moved and is not in the swagger doc anymore